proto/outbound.proto

```diff
@@ -436,3 +444,15 @@ message ExponentialBackoff {
   // Must be greater than or equal to 0.0.
   float jitter_ratio = 3;
 }
+
+// Error type that is used to indicate that any traffic
+// that is delivered through a route should be failed.
+message RouteError {
+  string message = 1;
+}
+
+// Error type that is used to indicate that any traffic
+// that targets a particular backend should be failed.
+message BackendError {
```

Comment:
I'd be inclined to move this as a submessage of RouteBackend, where it's actually used, and perhaps call it something like Invalid, to make it clear that this is a failure used to indicate an illegal backend configuration.